Modules/Common/include/Common/TH2Ratio.h

```diff
@@ -70,11 +70,13 @@ class TH2Ratio : public T, public o2::mergers::MergeInterface
   Bool_t Add(const TH1* h1, const TH1* h2, Double_t c1 = 1, Double_t c2 = 1) override;
   Bool_t Add(const TH1* h1, Double_t c1 = 1) override;
   void SetBins(Int_t nx, Double_t xmin, Double_t xmax, Int_t ny, Double_t ymin, Double_t ymax) override;
+  void Sumw2(Bool_t flag = kTRUE) override;
```

Comment:
```suggestion
  void SetSumw2(Bool_t flag = kTRUE) override;
```

Reply:
For consistency with other setters, should we make is `SetSumw2`?

Reply:
@knopers8 here I opted for overriding the virtual [TH1::Sumw2()](https://root.cern.ch/doc/master/classTH1.html#aefa4ee94f053ec3d217f3223b01fa014) function, since thre THxRatio classes are designed to behave as much as possible like standard histograms.

Reply:
ah, thanks, i didn't realise this is an override. all clear!